storage/src/tests/persistence/common/persistenceproviderwrapper.h

```diff
@@ -50,8 +51,10 @@ class PersistenceProviderWrapper : public spi::AbstractPersistenceProvider
 private:
     spi::PersistenceProvider& _spi;
     spi::Result _result;
+    mutable std::mutex  _lock;
     mutable std::vector<std::string> _log;
     uint32_t _failureMask;
+    using Guard = std::unique_lock<std::mutex>;
```

Comment:
std::lock_guard&lt;std::mutex&gt; should be sufficient here.

Reply:
Thanks, fixed